src/server/model/event.js

```diff
@@ -23,7 +23,7 @@ const mongoose = require("mongoose"),
             "required": true
         },
         "isActive": {
-            "type": Boolean,
+            "type": [String],

```

Comment:
Why is this an array string? @KrastanD

Reply:
Because it will be an array of usernames that have marked it inactive. Front end will display an event if that list is less than 3 in length.

Reply:
Ok got it, I remember now